content/en/docs/concepts/glossary.md

```diff
@@ -29,7 +29,7 @@ One or more [`Services`](#service) designed for end users or other applications.
 
 ### **APM**
 
-Application Performance Monitoring. Typically a back-end of the
+Application Performance Monitoring is about monitoring software applications, their performance (speed, reliability, availability, etc.) to detect issues, alerting and tooling for finding the root cause. Typically a back-end of the
```

Comment:
That's a good definition. Ty.

`Typically a back-end of the [`Tracing`](#trace) [`Data Source`](#data-source).`

I still don't like that sentence, because it is misleading and I hear people saying "APM is only for traces", can you remove it.

Reply:
sure let me remove these lines from the document.

Reply:
I removed the lines from the document. Could you please review once.